framework/tests/src/tests/ValueIgnoreRangeOverflowTest.java

```diff
@@ -0,0 +1,35 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.common.value.ValueChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Tests the constant value propagation type system without overflow.
+ *
+ * <p>NOTE: $CHECKERFRAMEWORK/framework/tests/value/ needs to be on the classpath. Otherwise
```

Comment:
Where is this done? I don't see a classpath option here or in framework/build.xml .
If you only need this for one file, you should make it a jtreg test.